### DISCUSSION TITLE
Swagger

### DISCUSSION THREAD

**Discussion by andi-faizal94** [honojs/hono#3742] [Q&A]
how to integrate swagger with hono?

i use 
app.get("/ui", swaggerUI({ url: "/doc" }));

but this is error like this
<img width="1472" alt="Screenshot 2024-12-12 at 09 23 36" src="https://github.com/user-attachments/assets/7e46af28-9d17-4f75-b75d-fab1aa43ee82" />

**Comment by jesielviana** [comment]
You shoud create a route `/doc`.

See this example: [https://github.com/jesielviana/hono-api/blob/main/src/app.ts](https://github.com/jesielviana/hono-api/blob/main/src/app.ts)

**Comment by ArjixWasTaken** [comment]
by reading the darn guide
https://github.com/honojs/middleware/tree/main/packages/swagger-ui#with-openapihono-usage
